components/eamxx/tests/coupled/physics_only/shoc_p3_nudging/CMakeLists.txt

```diff
@@ -26,7 +29,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/output_remapped.yaml
                ${CMAKE_CURRENT_BINARY_DIR}/output_source_data_remapped.yaml)
 CreateUnitTestFromExec (shoc_p3_source shoc_p3_nudging
       EXE_ARGS "--use-colour no --ekat-test-params ifile=input_source_data.yaml"
-      PROPERTIES FIXTURES_SETUP source_data FIXTURES_REQUIRED create_vertical_remap_file)
+      PROPERTIES FIXTURES_SETUP source_data 
+      FIXTURES_REQUIRED create_vertical_remap_file 
+      FIXTURES_REQUIRED create_nudging_weights_file)
```

Comment:
This won't work. Passing the same property twice causes the second instance to override the first one. To set multiple fixtures, you should do this:
```
  CreateUnitTestFromExec (....
     FIXTURES_REQUIRED "create_vertical_remap_file;create_nudging_weights_file")
```

Reply:
but I have this error, when I use FIXTURES_REQUIRED "create_vertical_remap_file;create_nudging_weights_file")

CMake Error at /gpfs/jlse-fs0/projects/climate/xyuan/scream.nudging/externals/ekat/cmake/EkatCreateUnitTest.cmake:375 (set_tests_properties):
  set_tests_properties called with incorrect number of arguments.
Call Stack (most recent call first):
  cmake/ScreamUtils.cmake:130 (EkatCreateUnitTestFromExec)
  tests/coupled/physics_only/shoc_p3_nudging/CMakeLists.txt:30 (CreateUnitTestFromExec)

Reply:
Ah, yes, we have some issue with passing some properties that are lists. I tried this with `add_test`, but not with the Ekat utility. Fortunately, we have a special handling for FIXTURES...but unfortunately there is a bug in it. If you can hold this PR for a bit more, I can fix it in Ekat, then update ekat in scream repo?

Reply:
sure, please go ahead, it is possibly the reason that the cpu test failed, it shows _"Failed test depencies: shoc_p3_source"_

Reply:
The PR was merged in ekat. #2586 is undergoing testing right now. Once merged, we can re-test this, and should hopefully pass.